locales/ja/messages.json

```
@@ -0,0 +1,127 @@
+  "extensionDescription": {
+    "message": "ウェブページ上のテキストの間隔に関するプロパティを変更するツール。WCAGに準拠するための情報も提供。"
+  },
+  "toggleCommand": {
+    "message": "テキストのプロパティを有効または無効にする。",
+    "description": "ブラウザでのキーボードショートカットの説明"
+  },
+  "buttonSetWCAGValues": {
+    "message": "WCAGの基準値を設定"
+  },
+  "buttonResetAllValues": {
+    "message": "全ての値をリセット"
+  },
+  "valueDefault": {
+    "message": "デフォルト",
+    "description": "テキストのデフォルトのプロパティ値"
+  },
+  "lineHeight": {
+    "message": "行の間隔（行送り）"
+  },
+  "letterSpacing": {
+    "message": "文字の間隔（文字送り）"
+  },
+  "wordSpacing": {
+    "message": "単語の間隔"
+  },
+  "paragraphSpacing": {
+    "message": "段落に続く間隔"
+  },
+  "active": {
+    "message": "実行"
+  },
+  "decreasePropertyValue": {
+    "message": "$PROPERTY$ の値を下げる",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "increasePropertyValue": {
+    "message": "$PROPERTY$ の値を上げる",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "resetPropertyValue": {
+    "message": "$PROPERTY$ の値をリセット",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "propertiesWereEnabledDisabled": {
+    "message": "カスタムのテキストプロパティ値を $STATUS$ に設定しました。",
+    "placeholders": {
+      "status": {
+        "content": "$1"
+      }
+    }
+  },
+  "enabled": {
+    "message": "有効"
+  },
+  "disabled": {
+    "message": "無効"
+  },
+  "propertiesWereSetToWCAG": {
+    "message": "テキストのプロパティ値を WCAG の基準値に設定しました。"
+  },
+  "propertiesWereResetToDefaults": {
+    "message": "テキストのプロパティ値をデフォルトに戻しました。"
+  },
+  "propertyWasResetToDefault": {
+    "message": "$PROPERTY$ をデフォルト値に戻しました。",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "instructionsForUsage": {
+    "message": "Text Spacing Editorの使い方は？"
+  },
+  "instructionsForActive": {
+    "message": "上部にある<strong class='hint'>実行</strong>スイッチを使用して、テキストの間隔に関するプロパティ値及びこの拡張機能のすべてのインタラクティブなコントロールを有効または無効にすることができます。また、ショートカットキー <code class='hint'>Ctrl+Shift+F</code>（Macでは <code class='hint'>Command+Shift+F</code>）を使うこともできます（ブラウザの拡張機能設定で、お好みのショートカットキーを設定することも可能）。 プロパティのカスタム設定を切り替えることができるので、チェック作業やカスタム設定の保存に便利です。"
+  },
+  "instructionsForWCAGButton": {
+    "message": "すべてのテキスト間隔のプロパティを<a href='#wcag-conformance-values'>WCAGに準拠した値</a>に設定するには、<strong class='hint'>「WCAGの基準値を設定」</strong>ボタンを使ってください。"
+  },
+  "instructionsForResetAllButton": {
+    "message": "すべてのテキストのプロパティをデフォルト値に戻すには、<strong class='hint'>「全ての値をリセット」</strong>ボタンを使います。"
+  },
+  "instructionsForSliders": {
+    "message": "それぞれのテキスト間隔プロパティの値を設定するには、<strong class='hint'>スライダー</strong>または <strong class='hint'><span class='visually-hidden'>下げる</span><svg class='icon icon--inline' viewBox='0 0 24 24' ><path d='M1 9 13H5v-2h14v2z'></path></svg></strong> と <strong class='hint'><span class='visually-hidden'>上げる</span><svg class='icon icon--inline' viewBox='0 0 24 24' ></path></svg></strong> ボタン（長押しイベントをサポート）を使ってください"
```

Here just small fix of icons:

```suggestion
    "message": "それぞれのテキスト間隔プロパティの値を設定するには、<strong class='hint'>スライダー</strong>または <strong class='hint'><span class='visually-hidden'>下げる</span><svg class='icon icon--inline' viewBox='0 0 24 24' ><path d='M19 13H5v-2h14v2z'></path></svg></strong> と <strong class='hint'><span class='visually-hidden'>上げる</span><svg class='icon icon--inline' viewBox='0 0 24 24'><path d='M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z'></path></svg></strong> ボタン（長押しイベントをサポート）を使ってください"
```